src/app/pages/home/components/onboarding-steps-list.tsx

```diff
@@ -0,0 +1,131 @@
+    imageFull: AddFundsFull,
+    imagePopup: AddFundsPopup,
+    route: RouteUrls.Buy,
+    title: OnboardingSteps.AddFunds,
+  },
+  {
+    body: 'Register your unique decentralized name',
+    imageFull: RegisterNameFull,
+    imagePopup: RegisterNamePopup,
+    route: 'https://btc.us/',
+    title: OnboardingSteps.RegisterName,
+  },
+  {
+    body: 'Try Stacks apps for finance, NFTs, blogging and more',
+    imageFull: ExploreAppsFull,
+    imagePopup: ExploreAppsPopup,
+    route: 'https://www.stacks.co/explore/discover-apps#apps',
+    title: OnboardingSteps.ExploreApps,
+  },
+];
+
+export function OnboardingStepsList() {
+  const navigate = useNavigate();
+  const [, setHideOnboardingSteps] = useHideOnboardingStepsState();
+  const [onboardingStepsStatus, setOnboardingStepsStatus] = useOnboardingStepsStatusState();
+  useOnboardingSteps();
+
+  const handleClick = useCallback(
+    (route: string, step: string) => {
+      if (step === OnboardingSteps.ExploreApps) {
+        setOnboardingStepsStatus({
+          ...onboardingStepsStatus,
+          [OnboardingSteps.ExploreApps]: OnboardingStepStatus.Done,
+        });
+      }
+      if (route.startsWith('/')) navigate(route);
+      else openInNewTab(route);
+    },
+    [navigate, onboardingStepsStatus, setOnboardingStepsStatus]
+  );
+
+  return (
+    <>
+      <SpaceBetween
+        maxWidth={['unset', HOME_FULL_PAGE_MAX_WIDTH]}
+        mt={['base', 'base-loose']}
+        px={['base-loose', 'unset']}
+        width="100%"
+      >
+        <Stack spacing="tight" width="100%">
+          <Text fontSize={[1, 2]}>Welcome to Stacks 👋</Text>
+          <Title fontSize={[3, 4]}>Next steps for you</Title>
+        </Stack>
+        <Circle
+          data-testid={OnboardingSelectors.HideStepsBtn}
+          border="1px solid"
+          borderColor={color('border')}
+          color={color('bg')}
+          onClick={() => setHideOnboardingSteps(true)}
+          size="32px"
+        >
+          <FiX color={color('text-caption')} size="20px" />
+        </Circle>
+      </SpaceBetween>
+      <Flex
+        alignItems="center"
+        borderBottom="3px solid"
+        borderBottomColor={color('bg-4')}
+        data-testid={OnboardingSelectors.StepsList}
+        flexWrap={['wrap', 'no-wrap']}
+        gap={['tight', 'unset']}
+        justifyContent={['start', 'center']}
+        pb="extra-loose"
+        px={['loose', 'unset']}
+        mt="extra-loose"
+        rowGap={['tight', 'unset']}
+        width="100%"
+      >
+        {onboardingSteps.map(step => (
+          <OnboardingStepItem
+            body={step.body}
+            imageFull={step.imageFull}
+            imagePopup={step.imagePopup}
+            isDone={onboardingStepsStatus[step.title] === OnboardingStepStatus.Done}
+            key={step.title}
+            onClick={() => handleClick(step.route, step.title)}
+            title={step.title}
+          />
+        ))}
```

Comment:
Some would disagree with me but here, imo I'd just duplicate the markup than loop through it. When it comes to functionality, it's good to DRY, but with content of 4 items, duplication is fine, and avoids assumptions that content only has these properties. What if, say, one step has a custom property. How would it be passed it without the others becoming aware of it?

No action needed, this is good, just explaining my thinking